ssh.go

```diff
@@ -41,7 +45,8 @@ var (
 				Render
 )
 
-func sshServer(ln net.Listener) {
+func SshServer(ln net.Listener, vm string) {
+	vmName = vm
```

Comment:
```suggestion
	# TODO(shayne): Don't use a package global.
	vmName = vm
```

Reply:
This was one of my biggest itches with doing this, but a quick fix until a better solution is in place. At the moment, the code wasn't expecting a module.

Reply:
```
// TODO(shayne): Don't use a package global.
```

;-)